src/index.html

```diff
@@ -7,8 +7,99 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="nav">
+      <div class="nav__wrapper">
+        <a href="#" class="logo">
+          <img
+          src="images/logo.png"
+          alt="Moyo"
+          class="logo__img"
+          >
```

Comment:
```suggestion
          <img
            src="images/logo.png"
            alt="Moyo"
            class="logo__img"
          >
```
Fix all cases
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/c89a0907-c3d8-48d9-9cf4-6437d8b4873c)